libraries/core_libs/consensus/include/pbft/step/step.hpp

```diff
@@ -95,11 +30,18 @@ class Step {
    */
   size_t placeVote_(blk_hash_t const& blockhash, PbftVoteTypes vote_type, uint64_t round);
```

Comment:
as this is major refactor, lets not use _ in funciton names anymore just because it was there before

Reply:
I guess when we have Step classes, these classes should not really specify vote_type ? If we are voting in propose step class, then this vote can be only of propose_type and nothing else